### PR TITLE
Unified minikubeIP handling

### PR DESCRIPTION
**Pull request by a-thaler**
**Description**

Changes proposed in this pull request:

- Unified minikubeIP property
- Have not adjusted installer scripts as they still will work
- cleaned up local config overwrites

### CODE REVIEW

File: installation/resources/installer-config-local.yaml.tpl [hunk 1]
```diff
@@ -35,6 +35,7 @@ data:
   global.isLocalEnv: "true"
   global.domainName: "kyma.local"
   global.adminPassword: ""
+  global.minikubeIP: ""
```
Comment (aszecowka) [inline installation/resources/installer-config-local.yaml.tpl]: probable we need to introduce appropriate changes to kyma-cli https://github.com/kyma-project/cli/blob/3fae2a44236192e7b41867c4554f072bdfc47247/pkg/kyma/cmd/install/cmd.go#L844 to set minikubeIP for `installation-config-overrides`

File: installation/resources/installer-config-local.yaml.tpl [hunk 2]
```diff
@@ -97,71 +98,6 @@ metadata:
...
-metadata:
-  name: core-overrides
-  namespace: kyma-installer
-  labels:
-    installer: overrides
-    component: core
-    kyma-project.io/installation: ""
-data:
-  test.acceptance.ui.minikubeIP: ""
-  test.acceptance.ui.logging.enabled: ""
```
[the remark this answers is not in the document]
Comment (a-thaler) [reply installation/resources/installer-config-local.yaml.tpl]: The only occurrence has the default value of "false", so only when set to "true" it has an effect. And the effect is test execution

File: installation/resources/installer-config-local.yaml.tpl (hunk 1 above)
Comment (a-thaler) [reply installation/resources/installer-config-local.yaml.tpl]: yes, I plan that to do in the same ticket as soon this PR is merged

[the remark this answers is not in the document]
Comment (a-thaler) [reply installation/resources/installer-config-local.yaml.tpl]: I only know https://github.com/kyma-project/community/blob/master/guidelines/coding-standards/helm.md, so I can add a section there.
However, people will copy/paste.. so let's see that it is used everywherer

**Pull request merged**